katsdpcontroller/sdpcontroller.py

```diff
@@ -1435,10 +1435,14 @@ def remove_product(product):
             product_config.validate(config_dict)
             config_dict = await product_config.update_from_sensors(config_dict)
             config_dict = product_config.normalise(config_dict)
-        except (ValueError, product_config.SensorFailure, jsonschema.ValidationError) as error:
+        except product_config.SensorFailure as error:
+            retmsg = "CAM error: {}".format(error)
```

Comment:
I know I asked for this, but are we being too blunt in calling this a CAM error :) Apart from katstore being completely unavailable or timeouts on sensor requests, what else could trigger SensorFailure ?

Reply:
Easiest answer to that is to search for it in product_config.py. I guess some of the cases (like the sensor not existing) could be due to CBF as well.

Reply:
OK - how about "Sensor error (possible CAM problem): {}" ?

Reply:
Maybe be a bit more helpful and say: "Could not get sensor DATA from CAM:", DATA is not clear but sure you can substitute.

Reply:
Final bikeshed attempt: "Error retrieving sensor data from CAM: {}"

Reply:
Done